futures-util/src/future/maybe_done.rs

```diff
@@ -44,8 +44,25 @@ pub fn maybe_done<Fut: Future>(future: Fut) -> MaybeDone<Fut> {
 }
 
 impl<Fut: Future> MaybeDone<Fut> {
+    /// Returns an optional mutable reference to the output of the future.
```

Comment:
``Returns an [`Option`] containing a mutable reference...``

This currently sounds like "optionally mutable"

Reply:
Done.